src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist

```diff
@@ -587,8 +585,6 @@ staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods minus java.util.Co
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods minus java.util.List java.lang.Iterable
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods minus java.util.List java.lang.Object
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods minus java.util.List java.util.Collection
-staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods minus java.util.List removeAt int
-staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods minus java.util.List removeLast
```

Comment:
This is only applicable since Groovy 2.5.0, but the correct entry would have been:

```
staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods removeLast java.util.List
```